Lets Do This/Controllers/Settings/LDTSettingsViewController.m

```diff
@@ -175,6 +176,14 @@ - (void)handleLogoutTap:(UITapGestureRecognizer *)recognizer {
     [self presentViewController:logoutAlertController animated:YES completion:nil];
 }
 
+- (void)presentUserConnectViewController {
```

Comment:
Minor, but i updated my comment to name this `pushUserConnectViewController` since we're actually calling `pushViewController` and not `presentViewController`. Might as well rename when going in to fix the if/else logic